.travis.yml

```diff
@@ -1,14 +1,20 @@
+os: linux
 language: java
-install: "./install.sh"
 jdk:
 - openjdk8
 jobs:
   include:
-  - stage: build
-    script: "./build.sh"
-branches:
-  only:
-  - master
-before_install:
-- openssl aes-256-cbc -K $encrypted_7bb25f4185a5_key -iv $encrypted_7bb25f4185a5_iv
-  -in keys.gpg.enc -out keys.gpg -d
+    install: skip
+    script:
+      - bash ./build.sh || travis_terminate 1;
+    before_deploy: 'openssl aes-256-cbc
+      -K $encrypted_7bb25f4185a5_key
+      -iv $encrypted_7bb25f4185a5_iv
+      -in keys.gpg.enc -out keys.gpg -d &&
+      gpg --import ./keys.gpg'
+    deploy:
+      provider: script
+      script: bash ./deploy.sh
+      on:
+        repo: sparebank1/troxy
+        branch: master
```

Comment:
Hele jobben kjøres i samme container, ikke sant? Dvs. output fra build.sh eksisterer når deploy.sh kjører? Mener jeg var borti noe tilsvarende der det ikke var tilfelle.

Reply:
Det tror jag ska stæmma. Brukar man dæremot "build stages" så ær ett de ny vm/container.

Det ær egentligen aningens sløsaktig att køra `mvn deploy` från `deploy.sh`, kanske man kunde ha kørt `mvn xxx:signing-sonatype  deploy:deploy` eller nåt åt det hållet.

Reply:
Ok. Da må det ha vært stages jeg drev og rotet med. 
Er vel ikke så farlig om man må en ekstra tur gjennom Maven. Kanskje sørge for at tester og andre ting som ikke er nødvendig for å få bygd artefakten, ikke kjøres i deploy-steget. :)

Reply:
Ær inte særskilt många tester i Troxy-kodebasen føreløpigt ;-)